### PR TITLE
Setup accessibility-alt-text-bot.yml workflow

### PR DESCRIPTION
**Pull request by kendallgassner**
### What

This workflow runs an action against issue/pr descriptions and comments reminding users to add alt text to their images.

Closes # (type the issue number after # if applicable; otherwise remove this line)

### Screenshots

Please provide before/after screenshots for any visual changes

### Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Changes are [SSR compatible](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#ssr-compatibility)
- [ ] Tested in Chrome
- [ ] Tested in Firefox
- [ ] Tested in Safari
- [ ] Tested in Edge

Take a look at the [What we look for in reviews](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#what-we-look-for-in-reviews) section of the contributing guidelines for more information on how we review PRs.

### CODE REVIEW

File: .changeset/four-ducks-fly.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```
Comment (kendallgassner) [inline .changeset/four-ducks-fly.md:2]: I don't think this should change the project version at all...

Comment (kendallgassner) [reply .changeset/four-ducks-fly.md:2]: Is that possible?

Comment (joshblack) [reply .changeset/four-ducks-fly.md:2]: Definitely! Feel free to remove the `.changeset` here and I can add the `skip changeset` label 👍

**Review by joshblack — APPROVED** [review]
Very cool! Excited to try it out 🥳

**Comment by kendallgassner** [comment]
@joshblack is it best to call this a patch?

**Review by TylerJDev — APPROVED** [review]
Thanks for adding this @kendallgassner! ✨

**Comment by TylerJDev** [comment]
> is it best to call this a patch?

I think Josh suggested removing the `.changeset` file entirely, as it might not need it with the change it's producing.

**Comment by joshblack** [comment]
@kendallgassner I totally agree with what you said in: https://github.com/primer/react/pull/3308#discussion_r1200688702 and would just remove the changeset. I added the `skip changeset` label so you shouldn't see any CI failures from the changeset workflow anymore!

**Pull request merged**